seq2science/rules/qc.smk

```diff
@@ -555,21 +620,26 @@ rule multiqc:
 
 
 def get_trimming_qc(sample):
-    if get_workflow() == "scatac_seq":
-        # we (at least for now) do not was fastqc for each single cell before and after trimming.
-        # still something to think about to add later, since that might be a good quality check though.
-        return expand(f"{{qc_dir}}/fastqc/{sample}_{{fqext}}_trimmed_fastqc.zip", **config)
-    else:
-        if config['layout'][sample] == 'SINGLE':
-            return expand([f"{{qc_dir}}/fastqc/{sample}_fastqc.zip",
-                           f"{{qc_dir}}/fastqc/{sample}_trimmed_fastqc.zip",
-                           f"{{qc_dir}}/trimming/{sample}.{{fqsuffix}}.gz_trimming_report.txt"],
-                           **config)
+    if config["trimmer"] == "trimgalore":
+        if get_workflow() == "scatac_seq":
+            # we (at least for now) do not was fastqc for each single cell before and after trimming.
```

Comment:
"we (at least for now) do not was fastqc"

so far this is the biggest issue I have been able to find :D

Reply:
I agree. Any proposals? The "problem" is that fastp does both qc and trimming, so not sure how to name it, but it is unclear indeed.

sidenote: get_trimming_qc needs cleanup, but I mentioned that before

Reply:
`# do not load all QC data for each single cell (its too much for MultiQC)`?

It doesn't clear up what we leave out, but the reason is stated.